lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -45,9 +57,98 @@
     seed_milestone_relationship_types,
 )
 
+from lms.djangoapps.ccx.models import CustomCourseForEdX
+
 # pylint: disable=protected-access
 
 
+class CoachAccessTestCaseCCX(SharedModuleStoreTestCase, LoginEnrollmentTestCase):
+    """
+    Test if user is coach on ccx.
+    """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Set up course for tests
+        """
+        super(CoachAccessTestCaseCCX, cls).setUpClass()
+        cls.course = CourseFactory.create()
+
+    def setUp(self):
+        """
+        Set up tests
+        """
+        super(CoachAccessTestCaseCCX, self).setUp()
+
+        # Create ccx coach account
+        self.coach = AdminFactory.create()
+        self.client.login(username=self.coach.username, password="test")
```

Comment:
It looks like "test" is the default password in that factory, but that's not clear from looking at this test. Can you make it clearer? I'd suggest indicating the password when you call AdminFactory:
```python
self.coach = AdminFactory.create(password="test")
self.client.login(username=self.coach.username, password="test")
```